src/auth.service.js

```diff
@@ -21,7 +19,7 @@
     var eventBus = $rootScope.$new(true);
 
     function getSession() {
-      var sessionUrl = options.sessionEndpoint;
+      var sessionUrl = options.url + options.sessionEndpoint;
```

Comment:
okay i opted for simplifying things as much as possible, thus i removed `url` from the code as well